roles/reset/tasks/main.yml

```diff
@@ -199,7 +199,7 @@
     - "{{ ansible_env.HOME | default('/root') }}/.helm"
     - "{{ etcd_data_dir }}"
     - /var/lib/etcd-events
```

Comment:
Maybe also update this one to `etcd_events_data_dir` ?

Reply:
@floryut ok！update this one also@~@